src/etl/presentation/api.py

```
@@ -34,6 +34,40 @@ class LoginRequest(BaseModel):
+    
+    health_status = {
+        "status": "healthy",
+        "components": {
+            "etl_service": "up",
+            "qdrant": "unknown"
+        }
+    }
+    
+    try:
+        # Проверяем, отвечает ли Qdrant по сети. 
+        # У qdrant_client метод get_locks() — самый быстрый способ пинга.
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `У` (CYRILLIC CAPITAL LETTER U). Did you mean `Y` (LATIN CAPITAL LETTER Y)?

(RUF003)